baggers.js

```diff
@@ -4860,11 +4860,11 @@ var data = {
             "since": "2014-02-14",
             "name": "Boris Schapira",
             "bio": "Consultant - Clever Age",
-            "picture": "http://avatars.borisschapira.com/SudWeb12_300.jpg",
+            "picture": "//avatars.borisschapira.com/boris_1light_300_fuchsia.jpg",
```

Comment:
Nope : if you do not precise it, the browser will use the current protocol. As I support both http and https on avatars.borisschapira.com, it ensure that my avatar will be load in https if the website, in the future, migrate to https.